packages/core/src/components/pages/auth/components/updatePassword/index.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from "react";
+
+import { useTranslate, useRouterContext, useUpdatePassword } from "@hooks";
+import { IAuthCommonProps, IUpdatePasswordForm } from "../..";
+
+export const UpdatePassword: React.FC<IAuthCommonProps> = ({
+    backLink,
+    updatePasswordLink,
+}) => {
+    const translate = useTranslate();
+    const { Link } = useRouterContext();
+
+    const { mutate: updatePassword } = useUpdatePassword<IUpdatePasswordForm>();
+
+    const [password, setPassword] = useState("");
+    const [newpPassword, setNewPassword] = useState("");
```

Comment:
```suggestion
    const [newPassword, setNewPassword] = useState("");
```
I am not sure the name of state but it has typo